README.md

```diff
@@ -83,23 +83,20 @@ When ignoreWhitespaces mode is on then /\s/, " ", "\t", "\n" won't work and you
 In ignoreWhitespaces mode these symbols check context and not consuming characters from input.
 For example sequence: 'a' newLine space space 'b' will match characters 'a' and 'b' separated by at least one space and at least one new line symbol
 
-##### text
```

Comment:
I believe this is a duplication of `##### "text"` or I didn't understand the intention?